From 19dd07dd329a69550fdb4b2582634c974db238bf Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 15 Jan 2019 17:41:20 +0200
Subject: [PATCH 0894/1051] arch64: dts: cn9130: enable usb2.0 by default

The usb2.0 from CP0 can be enabled by default for all configurations.
This is not true for USB3.0, which will be supported e.g. in setup B,
therefore remove generic phy description from common dtsi.

Change-Id: Ibd150b142d4dc2f3fab42711e989d40ca60d3f89
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/2158
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Ben Peled <Ben.Peled@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-db.dtsi | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
index b931e3d4da8f..129bb4aad77a 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
@@ -117,16 +117,14 @@
 };
 
 &cp0_usb3_0 {
+	status = "okay";
 	usb-phy = <&cp0_usb3_0_phy0>;
-	/* Generic PHY, providing serdes lanes */
-	phys = <&cp0_comphy1 0>;
 	phy-names = "usb";
 };
 
 &cp0_usb3_1 {
+	status = "okay";
 	usb-phy = <&cp0_usb3_0_phy1>;
-	/* Generic PHY, providing serdes lanes */
-	phys = <&cp0_comphy3 1>;
 	phy-names = "usb";
 };
 
-- 
2.17.1

